src/DynamoCoreWpf/Views/Core/AnnotationView.xaml

```diff
@@ -138,20 +138,46 @@
                 </Grid>
             </Border>
             <ControlTemplate.Triggers>
-                <!--Change the icon when toggled-->
-                <Trigger Property="IsChecked"
-                         Value="True">
+                <MultiDataTrigger>
+                    <MultiDataTrigger.Conditions>
+                        <Condition Binding="{Binding RelativeSource={RelativeSource Self}, Path=IsMouseOver}" Value="False"/>
+                        <Condition Binding="{Binding RelativeSource={RelativeSource Self}, Path=IsChecked}" Value="False"/>
```

Comment:
I thought `IsMouseOver` alone is good, what does the other check do?

Reply:
There are multiple conditions here: If the group is expanded we have one icon and a different hover icon corresponding to that, and if it is collapsed we have a different icon with a different hover icon.
So the icon depends on 2 states, that is the state of the group as well as mouse hover.